Owncloud iOs Client/Login/Login/UniversalLoginViewController.swift

```diff
@@ -538,53 +538,54 @@ public enum TextfieldType: String {
+                            )
                         )
-                    )
-                    print ("error getting information from URL")
-                    
-                } else if validatedURL != nil {
-                    
-                    self.setURLFooter(isType: .None)
-                    
-                    self.validatedServerURL = validatedURL;
-                    self.allAvailableAuthMethods = serverAuthenticationMethods as! [AuthenticationMethod]
-                    
-                    self.authMethodToLogin = DetectAuthenticationMethod.getAuthMethodToLoginFrom(availableAuthMethods: self.allAvailableAuthMethods)
-                    
-                    if (self.authMethodToLogin != .NONE) {
-                        self.setReconnectionButtons(hiddenStatus: true)
-
-                        if (self.authMethodToLogin == .BASIC_HTTP_AUTH) {
-                            self.textFieldURL.resignFirstResponder()
-                            self.textFieldUsername.becomeFirstResponder()
+                        print ("error getting information from URL")
+                        
+                    } else if validatedURL != nil {
+                        
+                        self.setURLFooter(isType: .None)
+                        
+                        self.validatedServerURL = validatedURL;
+                        self.allAvailableAuthMethods = serverAuthenticationMethods as! [AuthenticationMethod]
+                        
+                        self.authMethodToLogin = DetectAuthenticationMethod.getAuthMethodToLoginFrom(availableAuthMethods: self.allAvailableAuthMethods)
+                        
+                        if (self.authMethodToLogin != .NONE) {
+                            self.setReconnectionButtons(hiddenStatus: true)
                             
-                            if self.loginMode == .update {
-                                self.textFieldUsername.text = self.user?.username
-                                self.textFieldPassword.text = ""
+                            if (self.authMethodToLogin == .BASIC_HTTP_AUTH) {
+                                self.textFieldURL.resignFirstResponder()
+                                self.textFieldUsername.becomeFirstResponder()
+                                
+                                if self.loginMode == .update {
+                                    self.textFieldUsername.text = self.user?.username
+                                    self.textFieldPassword.text = ""
+                                }
+                            } else {
+                                self.setConnectButton(status: false)
+                                self.startAuthenticationWith(authMethod: self.authMethodToLogin)
                             }
+                            
+                            self.showURLSuccess(self.validatedServerURL.hasPrefix("https://"))
```

Comment:
Maybe this prefix should be coded in a constant and also this could be extracted into a function in the url utils to know if a url is secured with https or not

Reply:
👍